pyramid_oereb/lib/records/geometry.py

```diff
@@ -32,7 +32,7 @@ def __init__(
         self.law_status = law_status
         self.published_from = published_from
         self.geo_metadata = geo_metadata
-        if isinstance(geom, Point) or isinstance(geom, LineString) or isinstance(geom, Polygon):
+        if isinstance(geom, (Point, LineString, Polygon, MultiPolygon)):
```

Comment:
We have nearly all the geom type, then shouldn't be `BaseGeometry`?

Reply:
For me, it would be ok to use `BaseGeometry`.
@vvmruder and @voisardf, what do you think?

Reply:
Yes, MultiPolygon and MultiLineString are necessary - not sure about MultiPoint though. But in case of doubt, let's add that one too.
If I understand you right, you want to create a new BaseGeometry Class?

Reply:
No I speak about using this class https://github.com/Toblerity/Shapely/blob/b23ffb63dbcc4c622de0e42cbcbaf056fe6272fa/shapely/geometry/base.py#L184 :-)

Reply:
Since we have Point, Line, Polygon at this stage only we should clearly test for this. So -1 for general BaseType-Testing.